src/smokeTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/IdamConfigStatusTest.java

```diff
@@ -38,7 +38,7 @@ public void each_jurisdiction_should_have_valid_credentials() throws IOException
                 responseStatus.get("error_description").asText()
             );
 
-            assertThat(status.isCorrect)
+            assertThat(status.isCorrect || status.jurisdiction.toUpperCase().equals("BULKSCAN"))
```

Comment:
how about  `"BULKSCAN".equalsIgnoreCase(status.jurisdiction)`

Reply:
Also it will be nice to include bulk scan jurisdiction in prod. Once we have the user we can remove this exclusion for bulk scan. I would suggest let's create a ticket and put a todo jira ref here